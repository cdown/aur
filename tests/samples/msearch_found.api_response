{"version":1,"type":"msearch","resultcount":8,"results":[{"ID":218739,"Name":"mpdmenu","PackageBaseID":70444,"PackageBase":"mpdmenu","Version":"1.0.3-2","Description":"dmenu frontend to MPD","URL":"http:\/\/github.com\/cdown\/mpdmenu","NumVotes":3,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1369711771,"LastModified":1438966213,"License":"ISC","URLPath":"\/cgit\/aur.git\/snapshot\/mpdmenu.tar.gz","CategoryID":1,"Popularity":0},{"ID":218744,"Name":"xbanish","PackageBaseID":96833,"PackageBase":"xbanish","Version":"1.4-1","Description":"Hide the mouse cursor when typing","URL":"https:\/\/github.com\/jcs\/xbanish","NumVotes":1,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1438967392,"LastModified":1438967396,"License":"BSD","URLPath":"\/cgit\/aur.git\/snapshot\/xbanish.tar.gz","CategoryID":1,"Popularity":0},{"ID":218745,"Name":"xnviewmp","PackageBaseID":18641,"PackageBase":"xnviewmp","Version":"0.72-1","Description":"An efficient multimedia viewer, browser and converter.","URL":"http:\/\/www.xnview.com\/en\/xnviewmp\/","NumVotes":204,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1217012518,"LastModified":1438967782,"License":"custom","URLPath":"\/cgit\/aur.git\/snapshot\/xnviewmp.tar.gz","CategoryID":1,"Popularity":0},{"ID":218746,"Name":"rbenv","PackageBaseID":51956,"PackageBase":"rbenv","Version":"0.4.0-1","Description":"Simple Ruby version manager","URL":"https:\/\/github.com\/sstephenson\/rbenv","NumVotes":67,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1314619026,"LastModified":1438968186,"License":"MIT","URLPath":"\/cgit\/aur.git\/snapshot\/rbenv.tar.gz","CategoryID":1,"Popularity":0},{"ID":225910,"Name":"dwm-cdown-git","PackageBaseID":69703,"PackageBase":"dwm-cdown-git","Version":"0.r1747.7fb62b4-2","Description":"dwm with cdown's personalisations","URL":"https:\/\/github.com\/cdown\/dwm","NumVotes":1,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1367853025,"LastModified":1440719745,"License":"MIT","URLPath":"\/cgit\/aur.git\/snapshot\/dwm-cdown-git.tar.gz","CategoryID":1,"Popularity":0},{"ID":229275,"Name":"tzupdate","PackageBaseID":99473,"PackageBase":"tzupdate","Version":"0.4.0-2","Description":"Set the system timezone based on IP geolocation","URL":"http:\/\/github.com\/cdown\/tzupdate","NumVotes":1,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1441577921,"LastModified":1441747990,"License":"ISC","URLPath":"\/cgit\/aur.git\/snapshot\/tzupdate.tar.gz","CategoryID":1,"Popularity":0},{"ID":229289,"Name":"yturl","PackageBaseID":68930,"PackageBase":"yturl","Version":"1.20.0-2","Description":"YouTube videos on the command line","URL":"http:\/\/github.com\/cdown\/yturl","NumVotes":1,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1365426996,"LastModified":1441751184,"License":"MIT","URLPath":"\/cgit\/aur.git\/snapshot\/yturl.tar.gz","CategoryID":1,"Popularity":0},{"ID":230818,"Name":"clipmenu","PackageBaseID":96827,"PackageBase":"clipmenu","Version":"1.1.0-1","Description":"Clipboard management using dmenu","URL":"http:\/\/github.com\/cdown\/clipmenu","NumVotes":2,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1438964423,"LastModified":1442270409,"License":"ISC","URLPath":"\/cgit\/aur.git\/snapshot\/clipmenu.tar.gz","CategoryID":1,"Popularity":0}]}