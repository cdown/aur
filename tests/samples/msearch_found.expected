[
    {
        "category_id": 1,
        "description": "dmenu frontend to MPD",
        "first_submitted": 1369711771,
        "id": 218739,
        "last_modified": 1438966213,
        "license": "ISC",
        "maintainer": "cdown",
        "name": "mpdmenu",
        "num_votes": 3,
        "out_of_date": false,
        "package_base": "mpdmenu",
        "package_base_id": 70444,
        "popularity": 0,
        "url": "http://github.com/cdown/mpdmenu",
        "url_path": "/cgit/aur.git/snapshot/mpdmenu.tar.gz",
        "version": "1.0.3-2"
    },
    {
        "category_id": 1,
        "description": "Hide the mouse cursor when typing",
        "first_submitted": 1438967392,
        "id": 218744,
        "last_modified": 1438967396,
        "license": "BSD",
        "maintainer": "cdown",
        "name": "xbanish",
        "num_votes": 1,
        "out_of_date": false,
        "package_base": "xbanish",
        "package_base_id": 96833,
        "popularity": 0,
        "url": "https://github.com/jcs/xbanish",
        "url_path": "/cgit/aur.git/snapshot/xbanish.tar.gz",
        "version": "1.4-1"
    },
    {
        "category_id": 1,
        "description": "An efficient multimedia viewer, browser and converter.",
        "first_submitted": 1217012518,
        "id": 218745,
        "last_modified": 1438967782,
        "license": "custom",
        "maintainer": "cdown",
        "name": "xnviewmp",
        "num_votes": 204,
        "out_of_date": false,
        "package_base": "xnviewmp",
        "package_base_id": 18641,
        "popularity": 0,
        "url": "http://www.xnview.com/en/xnviewmp/",
        "url_path": "/cgit/aur.git/snapshot/xnviewmp.tar.gz",
        "version": "0.72-1"
    },
    {
        "category_id": 1,
        "description": "Simple Ruby version manager",
        "first_submitted": 1314619026,
        "id": 218746,
        "last_modified": 1438968186,
        "license": "MIT",
        "maintainer": "cdown",
        "name": "rbenv",
        "num_votes": 67,
        "out_of_date": false,
        "package_base": "rbenv",
        "package_base_id": 51956,
        "popularity": 0,
        "url": "https://github.com/sstephenson/rbenv",
        "url_path": "/cgit/aur.git/snapshot/rbenv.tar.gz",
        "version": "0.4.0-1"
    },
    {
        "category_id": 1,
        "description": "dwm with cdown's personalisations",
        "first_submitted": 1367853025,
        "id": 225910,
        "last_modified": 1440719745,
        "license": "MIT",
        "maintainer": "cdown",
        "name": "dwm-cdown-git",
        "num_votes": 1,
        "out_of_date": false,
        "package_base": "dwm-cdown-git",
        "package_base_id": 69703,
        "popularity": 0,
        "url": "https://github.com/cdown/dwm",
        "url_path": "/cgit/aur.git/snapshot/dwm-cdown-git.tar.gz",
        "version": "0.r1747.7fb62b4-2"
    },
    {
        "category_id": 1,
        "description": "Set the system timezone based on IP geolocation",
        "first_submitted": 1441577921,
        "id": 229275,
        "last_modified": 1441747990,
        "license": "ISC",
        "maintainer": "cdown",
        "name": "tzupdate",
        "num_votes": 1,
        "out_of_date": false,
        "package_base": "tzupdate",
        "package_base_id": 99473,
        "popularity": 0,
        "url": "http://github.com/cdown/tzupdate",
        "url_path": "/cgit/aur.git/snapshot/tzupdate.tar.gz",
        "version": "0.4.0-2"
    },
    {
        "category_id": 1,
        "description": "YouTube videos on the command line",
        "first_submitted": 1365426996,
        "id": 229289,
        "last_modified": 1441751184,
        "license": "MIT",
        "maintainer": "cdown",
        "name": "yturl",
        "num_votes": 1,
        "out_of_date": false,
        "package_base": "yturl",
        "package_base_id": 68930,
        "popularity": 0,
        "url": "http://github.com/cdown/yturl",
        "url_path": "/cgit/aur.git/snapshot/yturl.tar.gz",
        "version": "1.20.0-2"
    },
    {
        "category_id": 1,
        "description": "Clipboard management using dmenu",
        "first_submitted": 1438964423,
        "id": 230818,
        "last_modified": 1442270409,
        "license": "ISC",
        "maintainer": "cdown",
        "name": "clipmenu",
        "num_votes": 2,
        "out_of_date": false,
        "package_base": "clipmenu",
        "package_base_id": 96827,
        "popularity": 0,
        "url": "http://github.com/cdown/clipmenu",
        "url_path": "/cgit/aur.git/snapshot/clipmenu.tar.gz",
        "version": "1.1.0-1"
    }
]